net.minetest.Minetest.yaml

```diff
@@ -1,6 +1,7 @@
 app-id: net.minetest.Minetest
 runtime: org.freedesktop.Platform
-runtime-version: "20.08"
+runtime-version: "21.08"
+default-branch: stable
```

Comment:
```suggestion
```

This is not needed and actually more than annoying when developing.

Reply:
I think is more verbose declaring the branch